CONTRIBUTING.md

```diff
@@ -49,10 +49,22 @@ this will get rid of the errors.
 
     The errors should now be gone.
 
+### C) Setting up the JDT Language Server in JDT Language Server
+
+1) Please, download [Eclipse PDE support][1], open the `.vsix` file with a ZIP programm and extract the folder `/extension/server/` into a local directory.
+
+2) Ensure that [the initialize request][2] contains all paths to the extracted files in the `bundles` parameter.
```

Comment:
What is a good way to ensure that's the case? Is there additional documentation about that, for the case of nvim-jdtls, somewhere?

Reply:
https://github.com/emacs-lsp/lsp-java for emacs has a parameter "lsp-java-bundles - List of bundles that will be loaded in the JDT server." which is probably what is needed for emacs to work here! I haven't had time to try it yet.

Reply:
I could add links to the Neovim and Emacs configs as a reference.

Reply:
I have now tried it in emacs, and it works! Some things are a little bit slow, but that's perhaps because the java project I'm working on is large. This is great! Thanks so much @schrieveslaach , this has really made my week!

A template configuration for emacs would be something like:
"
2 b) For configuring PDE support in emacs, a configuration like the following can be used e.g. in your init.el:
```
(use-package lsp-java
  :config (setq lsp-java-bundles (directory-files "/path/to/extracted/folder/" t ".jar"))
  :hook (java-mode . lsp))
```

This will configure lsp-java to be activated when a java file is opened, and will send the required bundle files to JDTLS.
"